bullet/src/BulletDynamics/Dynamics/btDiscreteDynamicsWorld.h

```diff
@@ -229,6 +229,17 @@ ATTRIBUTE_ALIGNED16(class) btDiscreteDynamicsWorld : public btDynamicsWorld
 	{
 		return m_latencyMotionStateInterpolation;
 	}
+
+	// Contact callback support
+	void setContactAddedCallback(unsigned long callbackFunction) {
+		gContactAddedCallback = (ContactAddedCallback)callbackFunction;
```

Comment:
can we use an attribute for `gContactAddedCallback`, so we can avoid adding functions? it's best to avoid changes to bullet source files as much as possible.